src/appdev/ApplicationAPI.js

```diff
@@ -49,6 +50,14 @@ class ApplicationAPI {
         this.express.use(this.getPath() + version, router);
       });
     });
+
+    // // Setup Firebase Admin
+    console.log('here');
```

Comment:
```suggestion
    console.log('here');
```
Delete these console logs